src/refract/VisitorUtils.cc

```diff
@@ -0,0 +1,43 @@
+//
+//  refract/VisitorUtils.cc
+//  librefract
+//
+//  Created by Vilibald Wanča on 09/11/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+#include "Element.h"
+#include "Visitors.h"
+#include "VisitorUtils.h"
+namespace refract
+{
+
+    IElement* DefaultAttribute(const IElement& e) {
```

Comment:
`GetDefault` is a template function so I need to have a type, this function just gives me whatever there is as I don't care about the content I just need to get it if it is there.